### PR TITLE
Minor docs changes

### PR DESCRIPTION
**Pull request by allaway**
Updated a few links and some very minor changes to wording where I got confused. 

Also figured out how to add a Jira link that goes straight to [IT ticket creation](https://sagebionetworks.jira.com/jira/secure/CreateIssue.jspa?issuetype=3&pid=10083).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +26,13 @@ Message us in the [`#workflow_users`](https://sagebionetworks.slack.com/archives
 
 ## Tower User Onboarding
 
-Before you can use Nextflow Tower, you need to first deploy a Tower project, which consists an encrypted S3 bucket and the IAM resources (_i.e._ users, roles, and policies) that Tower requires to access the encrypted bucket and execute the workflow on [AWS Batch](https://help.tower.nf/compute-envs/aws-batch/). Once these resources exist, they need to be configured in Nextflow Tower, which is a process that has been automated using CI/CD.
+Before you can use Nextflow Tower, you need to first deploy a Tower project, which consists an encrypted S3 bucket and the IAM resources (_i.e._ users, roles, and policies) that Tower requires to access the encrypted bucket and execute the workflow on [AWS Batch](https://help.tower.nf/21.12/compute-envs/aws-batch/). Once these resources exist, they need to be configured in Nextflow Tower, which is a process that has been automated using CI/CD.
```
Comment (tthyer) [inline README.md:30]: I just noticed that there is a grammatical error here, not introduced by you, if you don't mind correcting it -- should read "which consists of."

File: README.md [hunk 2]
```diff
@@ -26,13 +26,13 @@ Message us in the [`#workflow_users`](https://sagebionetworks.slack.com/archives
...
 
-1. Determine what is known as the stack name by concatenating the project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
+1. Pick a 'stack name' by concatenating your project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
```
Comment (tthyer) [inline README.md:32]: I suggest instead "Create the 'stack name' ...". And then say something about how there is a naming convention and we're sticklers about it. 😁

**Review by tthyer — CHANGES_REQUESTED** [review]
Minor quibbles otherwise looks good! Thank you for improving the doc.

File: README.md [hunk 3]
```diff
@@ -26,13 +26,13 @@ Message us in the [`#workflow_users`](https://sagebionetworks.slack.com/archives
 
 ## Tower User Onboarding
 
-Before you can use Nextflow Tower, you need to first deploy a Tower project, which consists an encrypted S3 bucket and the IAM resources (_i.e._ users, roles, and policies) that Tower requires to access the encrypted bucket and execute the workflow on [AWS Batch](https://help.tower.nf/compute-envs/aws-batch/). Once these resources exist, they need to be configured in Nextflow Tower, which is a process that has been automated using CI/CD.
+Before you can use Nextflow Tower, you need to first deploy a Tower project, which consists an encrypted S3 bucket and the IAM resources (_i.e._ users, roles, and policies) that Tower requires to access the encrypted bucket and execute the workflow on [AWS Batch](https://help.tower.nf/21.12/compute-envs/aws-batch/). Once these resources exist, they need to be configured in Nextflow Tower, which is a process that has been automated using CI/CD.
```
Comment (allaway) [inline README.md:29]:
```suggestion
Before you can use Nextflow Tower, you need to first deploy a Tower project, which consists of an encrypted S3 bucket and the IAM resources (_i.e._ users, roles, and policies) that Tower requires to access the encrypted bucket and execute the workflow on [AWS Batch](https://help.tower.nf/21.12/compute-envs/aws-batch/). Once these resources exist, they need to be configured in Nextflow Tower, which is a process that has been automated using CI/CD.
```

File: README.md (hunk 1 above)
Comment (allaway) [reply README.md:30]: Sure thing!

File: README.md [hunk 4]
```diff
@@ -26,13 +26,13 @@ Message us in the [`#workflow_users`](https://sagebionetworks.slack.com/archives
...
 
-1. Determine what is known as the stack name by concatenating the project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
+1. Pick a 'stack name' by concatenating your project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
```
Comment (allaway) [inline README.md:31]:
```suggestion
1. Create a 'stack name' by following this naming convention: concatenate a project name with the suffix `-project` (_e.g._ `imcore-project`, `amp-ad-project`, `commonmind-project`).
```

**Review by tthyer — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (allaway) [reply README.md]: @tthyer - let me know what you think of this!

File: README.md (hunk 2 above)
Comment (allaway) [reply README.md:32]: Works for me! Made a change above - let me know what you think!

Comment (allaway) [reply README.md:32]: Whoops, somehow I lost my change...

Comment (allaway) [reply README.md:32]: Ah, I see, we made the change at the same time!

**Review by thomasyu888 — APPROVED** [review]

**Review by BrunoGrandePhD — APPROVED** [review]
🚀

**Pull request merged**